translatable_proc/src/data/config.rs

```diff
@@ -97,13 +102,29 @@ impl TranslatableConfig {
     }
 
     /// Get current seek mode strategy
-    pub fn seek_mode(&self) -> &SeekMode {
-        &self.seek_mode
+    pub fn seek_mode(&self) -> SeekMode {
+        self.seek_mode
     }
 
     /// Get current overlap resolution strategy
-    pub fn overlap(&self) -> &TranslationOverlap {
-        &self.overlap
+    pub fn overlap(&self) -> TranslationOverlap {
+        self.overlap
+    }
+
+    /// Set the locales path from a string (e.g., from environment variable)
+    fn set_path(&mut self, path: String) {
```

Comment:
setters? I'd possibly rather two ways of constructing this instead of setting pre set values or something like that, prefer immutability that mutability.